backend/test/user_handler_test.py

```diff
@@ -91,9 +101,9 @@ def test_delete(self, verify_token):
         verify_token._mock_return_value = {'email': self.other_user.email[0]}
         # check the user properties before delete it
         self.assertEquals(self.other_user.state, "active", "The user state should be 'active'")
-        self.assertEquals(self.other_user.institutions, [self.institution.key], 
+        self.assertEquals(self.other_user.institutions, [self.other_institution.key, self.institution.key], 
```

Comment:
check if the user is removed from institutions too. This test only check the institutions of user is updated but no check if the user is removed from member's institution